xcube/core/store/descriptor.py

```diff
@@ -119,6 +129,22 @@ class DatasetDescriptor(DataDescriptor):
     """
     A descriptor for a gridded, N-dimensional dataset represented by xarray.Dataset.
     Comprises a description of the data variables contained in the dataset.
+
+    :param data_id: An identifier for the data
+    :param type_specifier: A type specifier for the data
+    :param crs: A coordinate reference system identifier, as an EPSG, PROJ or WKT string
+    :param bbox: A bounding box of the data (order: min lon, min lat, max lon, max lat)
+    :param spatial_res: The spatial resolution of the data set
+    :param time_range: Start and end time delimiting this data's temporal extent
+    :param time_period: The data's periodicity if it is evenly temporally resolved.
```

Comment:
The time specification conventions are probably too lengthy to include in full here, but a "See data store conventions document" note might be useful.